tests/nlu/test_evaluation.py

```diff
@@ -394,28 +391,26 @@ async def test_eval_data(
         ],
     )
 
-    (_, _, persisted_path) = await train(
-        _config,
-        path=str(tmp_path),
-        data=data_importer,
-        component_builder=component_builder,
-        persist_nlu_training_data=True,
-    )
-
-    interpreter = Interpreter.load(persisted_path, component_builder)
+    with rasa.model.unpack_model(trained_simple_rasa_model) as unpacked_model_directory:
```

Comment:
how about having this as session wide fixture? Then we don't have to reload the model